.github/workflows/package-dependencies-update.yml

```diff
@@ -29,50 +28,46 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: '14' # Node version should be 14.
+          check-latest: true
 
       - name: Validate installed node version
-        continue-on-error: true # Continuing on error, adding this for debugging purposes.
         run: |
           echo "Using node version: $(node --version)"
           echo "Using npm version: $(npm -v)"
 
       - name: Install node modules package dependencies
-        continue-on-error: true # Continuing on error, adding this for debugging purposes.
         run: |
           npm install
 
       - name: Verify outdated node modules package dependencies versions
-        continue-on-error: true # Continuing on error, we need to check for outdated modules, so it's okay for this step to fail.
+        continue-on-error: true # Continuing on error. The initial command will return an exit 1 code; however, this is okay.
         run: |
           echo "Preparing to list outdated node modules versions within current build"
           npm outdated
 
       - name: Update node modules package dependencies versions within package.json
-        continue-on-error: true # Continuing on error, adding this for debugging purposes.
         run: |
           echo "Preparing to update all outdated package modules to latest versions"
-          npx npm-check-updates -u
+          echo "Ignoring package update for cypress"
+          npx npm-check-updates -u -x cypress
```

Comment:
Ignoring Cypress's latest version, there are currently issues with the latest versions.